InternetSemLimites/core/models.py

```diff
@@ -27,11 +27,16 @@ class Provider(models.Model):
     DISCUSSION = 'D'
     PUBLISHED = 'P'
     REFUSED = 'R'
+    EDIT = 'E'
+    OUTDATED = 'O'
+    ARCHIVED = 'A'
     STATUS = ((NEW, 'Aguardando moderação'),
               (DISCUSSION, 'Em discussão'),
               (PUBLISHED, 'Publicado'),
-              (REFUSED, 'Recusado'))
-
+              (REFUSED, 'Recusado'),
+              (EDIT, 'Edição aguardando moderação'),
+              (OUTDATED, 'Substituído por versão atualizada'),
+              (ARCHIVED, 'Arquivado'))
```

Comment:
_Arquivado_ seria sugestão de edição recusada, é isso? Se for, acho que podemos usar o `REFUSED` mesmo…